src/main/java/com/celements/cells/classes/CellsClassPackage.java

```diff
@@ -0,0 +1,34 @@
+package com.celements.cells.classes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.component.annotation.Requirement;
+
+import com.celements.model.classes.AbstractClassPackage;
+import com.celements.model.classes.ClassDefinition;
+import com.celements.web.classes.CelementsClassPackage;
+
+@Component(CelementsClassPackage.NAME)
+@ThreadSafe
+public final class CellsClassPackage extends AbstractClassPackage {
+
+  public static final String NAME = "celCellsClasses";

```

Comment:
for consistency, we never suffixed `Classes` so far in the package naming
```suggestion
  public static final String NAME = "celementsCells";
```

Reply:
```
#-# cellements class collections activated by default
celements.classcollections=celCellsClasses,celMenuClasses,celNavigationClasses,celCalendar,metaTag,blog,celPageTypeClasses,documentDetails,rteConfigClasses,oldCoreClasses,webSearch

```
IMHOP: The class-collection / class-package name must be retained in the refactoring. Otherwise activation of the given package in the XWikIPreferences of a database will not work anymore.

Reply:
packages are activated in `.properties` files with the key `celements.classdefinition.active`. this is completely independent from the old class collections. if you wan't to add legacy support via the `celements.classcollections` from `xwiki.cfg`, consider implementing `AbstractLegacyClassPackage` instead, there you'll have to implement `getLegacyName()` = `celCellsClasses`. the name for the package should then be of the existing scheme, e.g. `celementsCells`.

Reply:
👍 🥇